src/Breadcrumb/BreadcrumbLink.tsx

```diff
@@ -1,15 +1,30 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-export default function BreadcrumbLink({ as, clickHandler, linkProps }) {
+interface BreadcrumbLinkProps {
+  as: React.ElementType;
+  clickHandler?: (event: React.MouseEvent, link: any) => void;
+  linkProps: {
+    label: string;
```

Comment:
Thanks for these changes!

Could we accept a `ReactNode` here instead of string, so that we can add icons or update the design of the label text.

Example: 

<img width="419" height="93" alt="Image" src="https://github.com/user-attachments/assets/3e2dcab3-be9f-46e0-8a3b-9a42a32f291e" />

Reply:
That is a change that would need to be discussed and should not "sneak in" as part of a PR converting `.jsx`/`PropTypes` to `.tsx`/`TypeScript` types.

Reply:
Sure, I don't want to "sneak in" any changes :+1: